modules/common/react/index.ts

```diff
@@ -10,5 +10,6 @@ export * from './lib/parts/_brand-assets';
 export * from './lib/types';
 export * from './lib/genericStyles';
 export * from './lib/Popper';
+export * from './lib/theming';
 export {default as useTheme} from './lib/theming/useTheme';
```

Comment:
Nit: Can we just change these to named imports so they're caught by the line above?